src/components/forms/PaymentOptionsForm/PaymentOptionsForm.tsx

```diff
@@ -110,7 +124,6 @@ function PaymentOptionsForm({
             </Tabs.Trigger>
             <Tabs.Trigger
               value="credits"
-              disabled={true}
               className="flex gap-3 p-3 data-[state=active]:bg-foreground rounded border border-[#222224] data-[state=active]:border-grey text-white items-center flex-1 whitespace-nowrap transition-all duration-300 disabled:opacity-50"
             >
               <Tooltip
```

Comment:
<img width="477" alt="image" src="https://github.com/user-attachments/assets/ec8dc7a4-e41d-416f-a14e-39b797201492" />

This coming soon tooltip should be removed for this PR